contract/kernel/kernel.go

```diff
@@ -290,7 +290,7 @@ func (k *Kernel) validateUpdateIrreversibleSlideWindow(desc *contract.TxDesc) er
 		}
 		switch tp := desc.Args[argName].(type) {
```

Comment:
You can use the following form to verify the parameters.
``` go
if _, ok := desc.Args[argName].(float64); !ok {
    return fmt.Errorf("......")
}
```

Reply:
I get it.